### DISCUSSION TITLE
How to run automation based on node state? (dead/alive/asleep)

### DISCUSSION THREAD

**Discussion by peetipablo** [zwave-js/zwave-js#2053] [Q&A] [answered]
Hi All,

I moved to zwave-js this week and loving it so far, it's really stable and is working great.  Nice work!

I have one quick question.  Both my refrigerator and freezer are plugged into zwave outlets.  I had some automations that would notify me if the zwave status of these devices went to 'dead'.  The purpose of this is to notify me of either a loss of power or a circuit being tripped, so I can be made aware of the issue if I'm not home.

I'm not sure the best way to implement this now, since the devices are not listed like they were in zwave 1.4.  Below is a screenshot of my alert if it helps:

![image](https://user-images.githubusercontent.com/61809759/110996715-544bbb80-834a-11eb-9871-841d33cf3c93.png)

Thanks for any help you can provide.

**Comment by AlCalzone** [comment, marked as answer]
That's a question for the HomeAssistant repo. Please ask there.

**Reply by peetipablo** [reply]
No problem thank you.
